### PR TITLE
osd/ReplicatedPG: remove unneeded use of count

### PR DESCRIPTION
**Pull request by stiopaa1**
Signed-off-by: Michal Jarzabek stiopa@gmail.com

### CODE REVIEW

File: src/osd/ReplicatedPG.cc [hunk 1]
```diff
@@ -5936,7 +5937,8 @@ int ReplicatedPG::do_osd_ops(OpContext *ctx, vector<OSDOp>& ops)
 	for (map<string, pair<bufferlist, int> >::iterator i = assertions.begin();
 	     i != assertions.end();
 	     ++i) {
-	  bufferlist &bl = out.count(i->first) ? 
+	  auto out_entry = out.find(i->first);
+	  bufferlist &bl = (out_entry != out.end()) ?
```
Comment (liewegas) [inline src/osd/ReplicatedPG.cc]: since we don't reuse out_entry, i think the use of count() here is correct?

Comment (stiopaa1) [inline src/osd/ReplicatedPG.cc]: @liewegas 
I forgotten to use it in the next line, I have pushed the corrected commit.

Comment (liewegas) [inline src/osd/ReplicatedPG.cc]: Oh I see.  Thanks!

**Comment by tchaikov** [comment]
lgtm

**Comment by tchaikov** [comment]
- http://pulpito.ceph.com/kchai-2016-11-04_10:40:58-rados-wip-kefu-testing---basic-mira/
- http://pulpito.ceph.com/kchai-2016-11-05_15:35:54-rados-wip-kefu-testing---basic-mira/

failed tests are caused by #10049

**Pull request merged**